### DISCUSSION TITLE
대본 저장 API 관련 궁금한 점 있습니다

### DISCUSSION THREAD

**Discussion by silverpoodle** [meta-mingles/metamingle-server#14] [설계]
<img width="970" alt="image" src="https://github.com/meta-mingles/metamingle-server/assets/88484476/0ccde325-a830-4dbc-9ea7-7b293bc2e6f0">

위 순서로 진행될 경우 대본 저장은 영상 저장 시점과 같은 건가용?
Create 서비스를 어느 도메인에서 호출할지 고민입니다
@numerical43

**Comment by numerical43** [comment]
대본 저장이랑 영상저장 시점이 같아야하지않을까요? 만약에 저장 시점이 다르다면 대본 저장 버튼이 따로 있어야할텐데, 그렇게되면 사용성이 떨어질 것 같아요! 영상 업로드 확인을 클릭하면 대본이 저장된 후 영상을 저장하면 될 것 같은데... 확실히 도메인이 좀 애매하네요🤔 제 생각에는 버튼 클릭이 영상 업로드 관련이니까 숏폼이나 인터랙티브 무비 쪽에서 서비스를 호출하는게 맞지않나싶습니다!
